CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Changed
 
+- toolBar now shows now cut-off controls if window is too small #582
```

Comment:
https://media3.giphy.com/media/CiYImHHBivpAs/giphy.gif

Reply:
well the toolBar now shows now... ok i updated that ;)